user/ip-addresses.md

```diff
@@ -0,0 +1,18 @@
+---
+title: IP Addresses
+layout: en
+permalink: /user/ip-addresses/
+---
+
+For some it might be necessary to know the public IP addresses of build machines in order to whitelist them to access some internal resources. We run builds in a variety of different infrastructures, so which ones you use determines which IP ranges you need to whitelist.
```

Comment:
Knowing the IP addresses of the build machines Travis CI uses can be helpful when you need them whitelisted to access your internal resources. As builds are run in a variety of different infrastructures, the IP ranges to whitelist depend on the infrastructure your builds are running at: